streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/FlinkClientEndpoint.scala

```diff
@@ -84,6 +84,10 @@ object FlinkClientEndpoint {
         if (K8sFlinkConfig.isV2Enabled) KubernetesSessionClientV2.shutdown(request)
         else KubernetesNativeSessionClient.shutdown(request)
       }
+      case KUBERNETES_NATIVE_APPLICATION => {
+        if (K8sFlinkConfig.isV2Enabled) KubernetesApplicationClientV2.shutdown(request)
+        else _
```

Comment:
The "else _" here will return a runtime error, Modify in pattern matching to `case KUBERNETES_NATIVE_APPLICATION if K8sFlinkConfig.isV2Enabled`